tests/testthat/test-legacy_local.R

```diff
@@ -1,3 +1,5 @@
+skip_if_not_installed("filelock")
```

Comment:
FWIW our general policy is not to add skips: https://r-pkgs.org/dependencies-in-practice.html#sec-dependencies-in-suggests-in-tests

Reply:
Oh, interesting. I've run into problems with my tests using a suggested package failing on CRAN when the package gets archived. This kind of problem is partly why I/we/tidymodels starting doing a ["hard" `R CMD check`](https://github.com/rstudio/pins-r/blob/main/.github/workflows/R-CMD-check-hard.yaml), to avoid getting in that situation. Personally I've found it worthwhile for packages like vetiver, pins, broom, etc that provide methods for a lot of different packages. FWIW that GH action requires all the use of suggested packages in tests to be protected with skips.